commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommercePriceListAccountGroupTest.java

```diff
@@ -133,20 +234,26 @@ public void testPriceListWithUserAndDifferentRoleAccountGroups()
 
 		_userLocalService.addRoleUser(_role1.getRoleId(), _user);
 
+		CommerceAccountGroup commerceAccountGroup1 =
```

Comment:
Should be named `commerceAccountGroup` rather than `commerceAccountGroup1`, since there's no `commerceAccountGroup2`